Twig/NodeExpressionExtensionReference.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Copied from twigphp/twig.
+ *
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JMS\TranslationBundle\Twig;
+
+/**
+ * Represents an extension call node.
+ * Deprecated since twig 1.23 and removed in 2.0.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ */
+class NodeExpressionExtensionReference extends \Twig_Node_Expression
```

Comment:
@Nyholm @bdunogier How about implementing a node that replaces the whole `$transchoiceNode` ?

It would extend `Twig_Node_Expression_Call` and could be used both on Twig 1 and Twig 2.